### DISCUSSION TITLE
An error occurred, The application that is requesting authorization to use your Etsy account is not recognized.

### DISCUSSION THREAD

**Discussion by apzentral** [etsy/open-api#895] [Q&A]
Hi everyone,

When I follow with this document https://developer.etsy.com/documentation/tutorials/quickstart/. I could not get OAuth to work.

It always show me with this page,

![image](https://user-images.githubusercontent.com/3866678/222922323-e39c3268-e00e-4eca-ab27-92e9d3923806.png)

Does anyone know what could be the issue from this error?

Thank you so much.

**Comment by byter325** [comment]
@apzentral Hey, have you figured out a solution?
